website/static/js/registrationEditorExtensions.js

```diff
@@ -177,7 +177,16 @@ var Uploader = function(question) {
         question.value(question.formattedFileList());
     });
 
+    self.UPLOAD_LANGUAGE = 'You may attach up to 5 files to this question. You may attach files that you already have' +
+        'in this OSF project, or upload a new file from your computer. Uploaded files will automatically be added to this project' +
+        'so that they can be registered.';
+
     self.addFile = function(file) {
+        if(self.selectedFiles().length === 5) {
```

Comment:
The `length === 5` logic should probably be modified because it causes multiple error messages if more than 5 files are selected for upload:

![uploadfiles](https://cloud.githubusercontent.com/assets/7131985/14860827/8623a97c-0c78-11e6-8157-a1398111cab7.gif)